src/smart-components/role/add-role/schema.js

```diff
@@ -161,19 +161,62 @@ export const schemaBuilder = (container) => {
             name: 'add-permissions',
             title: intl.formatMessage(messages.addPermissions),
             StepTemplate: AddPermissionTemplate,
-            nextStep: ({ values }) =>
-              values &&
-              values['add-permissions-table'] &&
-              values['add-permissions-table'].some(({ uuid }) => uuid.split(':')[0].includes('cost-management'))
-                ? 'cost-resources-definition'
-                : 'review',
+            nextStep: ({ values }) => {
+              if (
+                values &&
+                values['add-permissions-table'] &&
+                values['add-permissions-table'].some(({ uuid }) => uuid.split(':')[0].includes('inventory'))
```

Comment:
You can simplify this by safe accessor
```suggestion
                values?.['add-permissions-table']?.some(({ uuid }) => uuid.split(':')[0].includes('inventory'))
```